i18n/messages/nl.json

```
@@ -151,6 +151,11 @@
     "isTranslated": true,
     "originalDefault": "CSS class"
   },
+  "3xnsUT": {
+    "defaultMessage": "Lijn interacties",
```

```suggestion
    "defaultMessage": "Lijn",
```
I feel like "interactions" is repeated way too much. I'm also inclined to rephrase this as "available drawing shapes" to make it simpler?

---

Yeah 'shapes' is probably better/easier to describe this that 'interactions'..